content/docs/import/import-from-csv.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Import data from CSV
+enableTableOfContents: true
+---
+
+This topic describes how to import data into a Neon database table from a CSV file.
+
+The instructions require a working installation of [psql](https://www.postgresql.org/download/). The `psql` client is the native command-line client for PostgreSQL. It provides an interactive session for sending commands to PostgreSQL. For more information about `psql`, refer to the [psql reference](https://www.postgresql.org/docs/current/app-psql.html), in the _PostgreSQL Documentation_.
+
+The following example uses the default `neondb` database, a table named `customer`, and a data file named `customer.csv`. Data is loaded from the `customer.csv` file into the `customer` table.
+
+1. Connect to the `neondb` database using `psql`. For example:
+
+   ```bash
+   psql postgres://casey:<password>@ep-polished-water-579720.us-east-2.aws.neon.tech/neondb
+   ```
+
+   <Admonition type="not">
+   For more information about connecting to Neon with `psql`, see [Connect with psql](../../connect/query-with-psql-editor).
+   </Admonition>
+
+2. Create the `customer` table.
+
+   ```sql
+   CREATE TABLE customer (
+     id SERIAL,
+     first_name VARCHAR(50),
+     last_name VARCHAR(50),
+     email VARCHAR(255),
+     PRIMARY KEY (id)
+   )
+   ```
+
+   <Admonition type="tip">
+   You can also create tables using the **SQL Editor** in the Neon Console. See [Query with Neon's SQL Editor](../../query-with-neon-sql-editor).
+   </Admonition>
+
+3. Prepare a `customer.csv` file with the following data:
+
+   ```text
+   First Name,Last Name,Email
+   Casey,Smith,casey.smith@example.com
+   Sally,Jones,sally.jones@example.com
```

Comment:
Not having the index at the start resulted in an error:
```bash
ERROR:  invalid input syntax for type integer: "Casey"
```

After adding the index, the command runs successfully and returns the number of records copied to the database
```bash
COPY 2
```
```suggestion
   1,Casey,Smith,casey.smith@example.com
   2,Sally,Jones,sally.jones@example.com
```